consensus/src/liveness/proposal_generator.rs

```diff
@@ -536,25 +548,17 @@ impl ProposalGenerator {
 
         let mut execution_backpressure_applied = false;
         if let Some(config) = &self.pipeline_backpressure_config.execution {
-            if pipeline_pending_latency.as_millis()
-                > config.back_pressure_pipeline_latency_limit_ms as u128
-            {
-                let execution_backpressure = self
-                    .pipeline_backpressure_config
-                    .get_execution_block_size_backoff(
-                        &self
-                            .block_store
-                            .get_recent_block_execution_times(config.num_blocks_to_look_at),
-                    );
-                if let Some(execution_backpressure_block_size) = execution_backpressure {
-                    values_max_block_txns_after_filtering.push(
-                        (execution_backpressure_block_size as f64
-                            * config.reordering_ovarpacking_factor.max(1.0))
-                            as u64,
-                    );
-                    values_max_txns_from_block_to_execute.push(execution_backpressure_block_size);
-                    execution_backpressure_applied = true;
```

Comment:
this is the fix - instead of updating both values_max_block_txns_after_filtering and values_max_txns_from_block_to_execute all the time (as values_max_txns_from_block_to_execute is not coordinated well with deduplication accounting), use only values_max_block_txns_after_filtering, and only if it is below 500, it will have to use values_max_txns_from_block_to_execute.